mantidproject/mantid
# Remove "Send cluster" in reduction UI

**Description of work.**

Remove *Send cluster* and related classes and methods in SNS reduction framework.


**To test:**

Launch MantidPlot on analysis cluster.  Check DGS reduction, Powder reduction and SANS reduction UIs.



Fixes #22950. 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
